### PR TITLE
Feature-6917 make ip assigned checkmark link to interface

### LINKED ISSUE #6917: Clickable link for IPAddress "assigned"
### NetBox version

v2.11.10 and v3.0-beta1

### Feature type

New functionality

### Proposed functionality

In a listing of IP addresses (`/ipam/ip-addresses/`), there is a column "Assigned" which has a green checkmark if it is assigned.

I would like this to be clickable to jump to the interface that it's assigned to - i.e. a link to `/dcim/interfaces/<N>/` or `/virtualization/interfaces/<N>/`

(Ideally I'd like it to show the device/VM name and interface name, but that would be a more expensive database query.  Making it link to the interface is essentially free)

### Use case

A quicker way to navigate to the assigned interface.  Otherwise, you have to first click on the IP address itself, which then shows its assigned device/VM and interface.

### Database changes

None

### External dependencies

None

### PR DESCRIPTION
**Pull request by royreznik**
### Fixes: #6917

Making assigned checkmark link to the interace in the ip table
Animation

### CODE REVIEW

**Comment by DanSheps** [comment]
Please update the changelog as well

**Comment by royreznik** [comment]
> Please update the changelog as well

What changelog? i only see realse-notes and there is no new one

File: netbox/ipam/tables/ip.py [hunk 1]
```diff
@@ -318,7 +318,8 @@ class IPAddressTable(BaseTable):
         verbose_name='NAT (Inside)'
     )
     assigned = BooleanColumn(
-        accessor='assigned_object_id',
+        linkify=True,
```
Comment (alon-david) [inline netbox/ipam/tables/ip.py:321]: change order of linkify and accessor so the variables will be sorted by alphabet order

File: docs/release-notes/version-3.0.md [hunk 2]
```diff
@@ -6,6 +6,7 @@
 
 * [#7118](https://github.com/netbox-community/netbox/issues/7118) - Render URL custom fields as hyperlinks in object tables
 * [#7323](https://github.com/netbox-community/netbox/issues/7323) - Add serial filter field for racks & devices
+* [#6917](https://github.com/netbox-community/netbox/issues/6917) - Make ip assigned checkmark in ip table link to interface
```
Comment (alon-david) [inline docs/release-notes/version-3.0.md:9]: pop this line on top so the issues will be ordered

**Pull request merged**